### PR TITLE
French translation Sept 2022

### PR DESCRIPTION
**Pull request by pierre-yves-monnet**

### CODE REVIEW

**Comment by RomanJRW** [comment]
Hey @pierre-yves-monnet 👋 
Just a quick one on this - the Optimize 3.9.0 translation files haven't been added yet, so the current ones here are liable to change over the coming weeks. It might make sense to run this again once we've updated it 🙂

**Comment by pierre-yves-monnet** [comment]
Hello Roman, of course, we (the translating team, ES, and IT are on board, too) will update this every time it's needed.
Could you publish a 'ping' in the slack channel #prj-translation-team ? Thank you.

**Comment by RomanJRW** [comment]
Yep, Optimize will notify the translation channel when this repo is "ready" to have it's translation files updated 👍

**Comment by RomanJRW** [comment]
@pierre-yves-monnet - This probably needs updating too and can then be reviewed and merged alongside the README table changes 🙂

File: README.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ Note that any language files that are missing a given key will cause Optimize to
...
 
-| Language | Version | Contributor   |
-|----------|---------|---------------|
-| German   | 3.9.0   | Optimize Team |
-| English  | 3.9.0   | Optimize Team |
+| Language      | Version | Contributor                                                     |
+|---------------|---------|-----------------------------------------------------------------|
+| German        | 3.9.0   | Optimize Team                                                   |
+| English       | 3.9.0   | Optimize Team                                                   |
+| French | 3.9.0   | Fullfifax (what's new) / pierre-yves-monnet (French dictionary) |
```
Comment (RomanJRW) [inline README.md:17]:
```suggestion
| French | 3.9.0   | @Fullfifax (what's new) / @pierre-yves-monnet (French dictionary) |
```

**Review by RomanJRW — APPROVED** [review]
Can't check the language but the changes LGTM 👍

**Pull request merged**